apps/breethe_web/config/config.exs

```diff
@@ -35,6 +35,8 @@ config :mime, :types, %{
 config :ja_serializer, key_format: {:custom, JsonApiKeys, :camelize, :underscore}
 config :ja_serializer, type_format: {:custom, JsonApiKeys, :camelize}
 
+config :cors_plug, origin: ["http://breethe.app"]
```

Comment:
https

Reply:
We should also allow `http://dev.breethe.app` so we can use that for developing locally

Reply:
fixed